plugins/chat/lib/chat/channel_fetcher.rb

```diff
@@ -175,60 +175,40 @@ def self.secured_direct_message_channels(user_id, guardian)
     end
 
     def self.secured_direct_message_channels_search(user_id, guardian, options = {})
```

Comment:
This is a big part of the change. We now want to be able to find direct message channels based on:
- name
- slug
- members usernames

Reply:
Note for myself, I have yet to write a spec for this